docs/fundamentals/dashboard/configuration.md

```diff
@@ -0,0 +1,117 @@
+---
+title: .NET Aspire dashboard configuration
+description: .NET Aspire dashboard configuration options
+ms.date: 03/13/2024
+ms.topic: reference
+---
+
+# Dashboard configuration
+
+The dashboard is configured when it starts up. Configuration includes frontend and OTLP addresses, the resource service endpoint, authentication, telemetry limits and more.
+
+If the dashboard is launched by the Aspire AppHost project then it is automatically configured to display the app's resources and telemetry. Configuration is provided when launching the dashboard in [standalone mode](overview.md#standalone-mode).
+
+There are a number of ways to provide configuration:
+
+- Command line arguments.
+- Environment variables. The `:` delimiter should be replaced with double underscore (`__`) in environment variable names.
+- Optional JSON configuration file. The `DOTNET_DASHBOARD_CONFIG_FILE_PATH` setting can be used to specify a JSON configuration file.
+
+Example of configuring the dashboard when started from a Docker container:
+
+```bash
+docker run --rm -it -p 18888:18888 -p 4317:18889 -d --name aspire-dashboard \
+    -e DASHBOARD__TELEMETRYLIMITS__MAXLOGCOUNT='1000' \
+    -e DASHBOARD__TELEMETRYLIMITS__MAXTRACECOUNT='1000' \
+    -e DASHBOARD__TELEMETRYLIMITS__MAXMETRICSCOUNT='1000' \
+    mcr.microsoft.com/dotnet/nightly/aspire-dashboard:8.0.0-preview.6
+```
+
+Alternatively, configuration in JSON configuration file that is specified using `DOTNET_DASHBOARD_CONFIG_FILE_PATH`:
+
+```json
+{
+  "Dashboard": {
+    "TelemetryLimits": {
+      "MaxLogCount": 1000,
+      "MaxTraceCount": 1000,
+      "MaxMetricsCount": 1000
+    }
+  }
+}
+```
+
+### Common configuration
+
+| Option | Default Value | Description |
+| ------ | ------------- | ----------- |
+| `ASPNETCORE_URLS` | <http://localhost:18888> | One or more HTTP endpoints through which the dashboard frontend is served. The frontend endpoint is used to view the dashboard in a browser. |
+| `DOTNET_DASHBOARD_OTLP_ENDPOINT_URL` | <http://localhost:18889> | The OTLP endpoint. OTLP endpoint hosts an OTLP service and recevies telemetry. |
+| `DOTNET_DASHBOARD_UNSECURED_ALLOW_ANONYMOUS` | `false` | Configures the dashboard to not use authentication and accepts anonymous access. This setting is a shortcut to configuring `Dashboard:Frontend:AuthMode` and `Dashboard:Otlp:AuthMode` to `Unsecured`. |
+| `DOTNET_DASHBOARD_CONFIG_FILE_PATH` | `null` | The path for an JSON configuration file. If the dashboard is being run in a Docker container then this is the path to the configuration file in a mounted volume. This value is optional. |
+| `DOTNET_RESOURCE_SERVICE_ENDPOINT_URL` | `null` | The gRPC endpoint to which the dashboard connects for its data. If this value is unspecified, the dashboard shows telemetry data but no resource list or console logs. This setting is a shortcut to `Dashboard:ResourceServiceClient:Url`. |
```

Comment:
Point to the proto file in the repo?

Reply:
Not in configuration. But there could be a page that talks about the resource service in more detail that has it.